misk/src/main/kotlin/misk/environment/DeploymentModule.kt

```diff
@@ -11,4 +13,20 @@ class DeploymentModule(
     bind<Deployment>().toInstance(deployment)
     bind<Env>().toInstance(env)
   }
+
+  companion object {
+    /**
+     * Return a Module that binds a [Deployment] and [Env] for a test environment.
+     *
+     * Until [Environment] is deleted, this also installs an [EnvironmentModule]
+     */
+    fun forTesting(): Module {
+      return Modules.combine(
+          DeploymentModule(
+              deployment = Deployment(name = "testing", isTest = true),
+              env = Env("TESTING")),
+          EnvironmentModule(Environment.TESTING)
```

Comment:
This will go away at the end of the migration, once Misk components no longer depend on `Environment`

Reply:
i'd leave this comment as a TODO in the code

Reply:
The final step is deleting the `Environment` enum altogether. I figured the compiler would remind me of that TODO :)